src/rulesets/Base/base.jl

```diff
@@ -117,7 +118,7 @@ end
 
 function rrule(::typeof(*), x::Number, y::Number)
     function times_pullback(ΔΩ)
-        return (NO_FIELDS,  @thunk(ΔΩ * y'), @thunk(x' * ΔΩ))
+        return (NO_FIELDS,  @thunk(ΔΩ * y), @thunk(x * ΔΩ))
```

Comment:
this is functionally the same right? The `'` before was just a no-op

Reply:
On real numbers it was a noop,
on complex numbers it was AFAICT just wrong?

Reply:
oh that makes much more sense (i never remember to think about Complex...)

Reply:
I wish i could forget about complex

Reply:
I have added test which fail without this change